app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+    people: dict = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+
+    for person in people:
+        person_list.append(Person(person["name"], person["age"]))
+
+    for index, person in enumerate(people):
+        if "wife" in person and person["wife"]:
+            person_list[index].wife = Person.people[person["wife"]]
+        if "husband" in person and person["husband"]:
+            person_list[index].husband = Person.people[person["husband"]]
```

Comment:
It is bad practice to refer to the dictionary by key. Here you check whether it exists, but do you need it? You can use .get(), so you will increase the readability of the code and not get an error if the key does not exist.